src/main/java/org/opensearch/ad/AnomalyDetectorExtension.java

```diff
@@ -80,56 +80,7 @@
-import org.opensearch.ad.transport.ADJobParameterTransportAction;
-import org.opensearch.ad.transport.ADJobRunnerAction;
-import org.opensearch.ad.transport.ADJobRunnerTransportAction;
-import org.opensearch.ad.transport.ADResultBulkAction;
-import org.opensearch.ad.transport.ADResultBulkTransportAction;
-import org.opensearch.ad.transport.ADStatsNodesAction;
-import org.opensearch.ad.transport.ADStatsNodesTransportAction;
-import org.opensearch.ad.transport.AnomalyDetectorJobAction;
-import org.opensearch.ad.transport.AnomalyDetectorJobTransportAction;
-import org.opensearch.ad.transport.AnomalyResultAction;
-import org.opensearch.ad.transport.AnomalyResultTransportAction;
-import org.opensearch.ad.transport.DeleteAnomalyDetectorAction;
-import org.opensearch.ad.transport.DeleteAnomalyDetectorTransportAction;
-import org.opensearch.ad.transport.DeleteModelAction;
-import org.opensearch.ad.transport.DeleteModelTransportAction;
-import org.opensearch.ad.transport.EntityResultAction;
-import org.opensearch.ad.transport.EntityResultTransportAction;
-import org.opensearch.ad.transport.ForwardADTaskAction;
-import org.opensearch.ad.transport.ForwardADTaskTransportAction;
-import org.opensearch.ad.transport.GetAnomalyDetectorAction;
-import org.opensearch.ad.transport.GetAnomalyDetectorTransportAction;
-import org.opensearch.ad.transport.IndexAnomalyDetectorAction;
-import org.opensearch.ad.transport.IndexAnomalyDetectorTransportAction;
-import org.opensearch.ad.transport.PreviewAnomalyDetectorAction;
-import org.opensearch.ad.transport.PreviewAnomalyDetectorTransportAction;
-import org.opensearch.ad.transport.ProfileAction;
-import org.opensearch.ad.transport.ProfileTransportAction;
-import org.opensearch.ad.transport.RCFResultAction;
-import org.opensearch.ad.transport.RCFResultTransportAction;
-import org.opensearch.ad.transport.SearchADTasksAction;
-import org.opensearch.ad.transport.SearchADTasksTransportAction;
-import org.opensearch.ad.transport.SearchAnomalyDetectorAction;
-import org.opensearch.ad.transport.SearchAnomalyDetectorInfoAction;
-import org.opensearch.ad.transport.SearchAnomalyDetectorInfoTransportAction;
-import org.opensearch.ad.transport.SearchAnomalyDetectorTransportAction;
-import org.opensearch.ad.transport.SearchAnomalyResultAction;
-import org.opensearch.ad.transport.SearchAnomalyResultTransportAction;
-import org.opensearch.ad.transport.SearchTopAnomalyResultAction;
-import org.opensearch.ad.transport.SearchTopAnomalyResultTransportAction;
-import org.opensearch.ad.transport.StatsAnomalyDetectorAction;
-import org.opensearch.ad.transport.StatsAnomalyDetectorTransportAction;
-import org.opensearch.ad.transport.StopDetectorAction;
-import org.opensearch.ad.transport.StopDetectorTransportAction;
-import org.opensearch.ad.transport.ValidateAnomalyDetectorAction;
-import org.opensearch.ad.transport.ValidateAnomalyDetectorTransportAction;
+import org.opensearch.ad.transport.*;
```

Comment:
Pasting the usual comment: [1](https://github.com/opensearch-project/anomaly-detection/pull/857#discussion_r1162017141), [2](https://github.com/opensearch-project/anomaly-detection/pull/857#discussion_r1162020980)

Reply:
Took care off